src/parser/jobs/blm/modules/ABC.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+
+import Module from 'parser/core/Module'
+import {i18nMark, Trans} from '@lingui/react'
+import {Table, Button} from 'semantic-ui-react'
+import DISPLAY_ORDER from './DISPLAY_ORDER'
+
+//value to be added to the gcd to avoid false positives
+const GCD_ERROR_OFFSET = 100
+/*since we look at end of a cast => beginning of a new one, we need to account for the time that's still on the gcd since we
+can literally not cast in that period. 1s is too much, but better than substracting not enough*/
+const DURATION_ERROR_OFFSET = 1000
+
+export default class AlwaysBeCasting extends Module {
+	static handle = 'blackmage_abc'
```

Comment:
Avoid prefixing handles. If it'd cause a conflict with another module, consider if it's been appropriately named. Is this ABC, or is it a report on when you _weren't_ ABC-ing?

Don't ask me for name suggestions tho I'm hopeless at that shit <img title=":_ notlikeblob :" src="https://raw.githubusercontent.com/BlobEmoji/blobs.gg/master/assets/blobs/notlikeblob.png" height="20">

Reply:
called it "NotCasting". I'll probably name my children "kid1", "kid2", ... someday